_projects/2023/100507931/100507931.Rmd

```diff
@@ -0,0 +1,385 @@
+                                "Cons-Moderate Dem/Lean Dem" = "#82A6BF",
+                                "Moderate-Lib Rep/Lean Rep" = "#E37F73"),
+                     
+                     breaks = c("Conservative Rep/Lean Rep", 
+                                "Moderate-Lib Rep/Lean Rep", 
+                                "Cons-Moderate Dem/Lean Dem", 
+                                "Liberal Dem/Lean Dem"),
+
+                     name= "") +
+  
+  scale_y_continuous(limits = c(0, 100), 
+                     breaks = seq(0, 100,by = 20),
+                     expand = expansion(0)) + #Y axis
+  
+ scale_x_date(
+    date_labels = "%Y",
+    limits = c(as.Date("1972-10-15"), max(datos$Date)),
+    breaks = seq(as.Date("1975-01-01"), max(datos$Date), by = "5 years"),
+    expand = expansion(0)) + 
+  
+ geom_vline(xintercept = as.Date("1972-10-15"), 
+            color = "darkgrey", size =0.5) +
+
+   
+   labs(title = "Trust in government by party and ideology",
+         
+       subtitle = "% who say they trust the government to do what is right just about always/most of the time",
+       caption = "PEW RESEARCH CENTER",
+       
+       x = "",
+       y = "") 
+ 
+ Plot
+```
+
+#### Annotations
+
+Now we also need to create a vector of date strings for the white area of the background to subsequently insert the annotations of the president names:
+
+```{r}
+white_xmin <- c("1972-10-15","1977-04-25","1989-01-15","2001-01-17","2017-04-11")
+
+white_xmax <- c("1974-12-01", "1980-11-07","1993-01-17","2008-12-21","2021-04-11")
+
+# Create the intervalos data frame
+white <- data.frame(xmin = as.Date(white_xmin),xmax = as.Date(white_xmax),
+                    intervalo = c("Nixon",  "Carter", "H.W.Bush", "W.Bush","Trump"))
+
+colores_white <- c("Nixon" = "#C81518","Carter" = "#2E5A87", "H.W.Bush" = "#C81518", "W.Bush" = "#C81518","Trump" = "#C81518")
+```
+
+Annotations for the white and the colored background area are added separately:
+
+```{r fig.height=5, fig.width=11}
+#Annotations in the colored intervals
+Plot <- Plot + annotate("text", x = as.Date(colored_xmin) + 
+                                (as.Date(colored_xmax) - 
+                                   as.Date(colored_xmin))/7, 
+                   
+                  y = 94, label = colored$intervalo, size = 4,
+                  angle = -90,  color = colores_intervalos[colored$intervalo])
+
+#Annotations in the white intervals
+Plot <- Plot + annotate("text", x = as.Date(white_xmin) + 
+                     (as.Date(white_xmax) - 
+                        as.Date(white_xmin))/7, 
+                   
+                  y = 94, label = white$intervalo, size = 4,
+                  angle = -90, color = colores_white[white$intervalo])
+
+Plot
+```
+
+### Arranging the final plot
+
+#### Google fonts
+
+```{r}
+library(showtext)
+
+sysfonts::font_add_google("Libre Franklin", family="franklin")
+showtext::showtext_auto()
```

Comment:
No need to call this line. Instead, you need to add `fig.showtext=TRUE` too all chunks with graphic output.